scylla/src/transport/downgrading_consistency_retry_policy.rs

```diff
@@ -49,11 +49,11 @@ impl Default for DowngradingConsistencyRetrySession {
 }
 
 impl RetrySession for DowngradingConsistencyRetrySession {
-    fn decide_should_retry(&mut self, query_info: QueryInfo) -> RetryDecision {
+    fn decide_should_retry(&mut self, query_info: RequestInfo) -> RetryDecision {
         let cl = match query_info.consistency {
             Consistency::Serial | Consistency::LocalSerial => {
                 return match query_info.error {
-                    QueryError::DbError(DbError::Unavailable { .. }, _) => {
+                    UserRequestError::DbError(DbError::Unavailable { .. }, _) => {
                         // JAVA-764: if the requested consistency level is serial, it means that the operation failed at
```

Comment:
🔧 You changed the type of the argument, but not the name. Could you change the name too (here and in other places)?